Dockerfile

```diff
@@ -24,20 +24,20 @@ RUN tar xzf epl.tgz --strip-components=1
 
 ENV NODE_ENV=production
 RUN cd src && \
-      npm install
+      npm install --no-progress --no-audit
 
 # Install extra plugins
-COPY package.json package.json
+COPY package.json package-lock.json /builder/
```

Comment:
`source` should be not a comma separated list ?

Reply:
I think both are accepted.

Reference: https://docs.docker.com/engine/reference/builder/#copy